tests/remote_point_evaluation/distributed_compute_intersection_locations_02.cc

```diff
@@ -0,0 +1,147 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2021 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+// FIle 3
+// check intersections are empty for J<1e-12 on all processors
+// File 4
+// ensure Convsersion to RPE is Correct
+
+#include "../tests.h"
```

Comment:
What is this test doing?

Reply:
Are you testing somewhere `distributed_compute_intersection_locations` or `RPE` set up by the new functions?

Reply:
`distributed_compute_intersection_locations` is tested in `tests/remote_point_evaluation/distributed_compute_intersection_locations_02.cc`. The setup of `RPE` is not tested directly, instead I test the conversion of `DistributedComputeIntersectionLocationsInternal` to `DistributedComputePointLocationsInternal` in `tests/remote_point_evaluation/convert_intersections_to_rpe_data_01.cc`. Is this OK or should I also set up `RPE` and compare it with an `RPE` instance which was setup with the points instead of `DistributedComputePointLocationsInternal`?

Reply:
>  should I also set up RPE and compare it with an RPE instance which was setup with the points instead of DistributedComputePointLocationsInternal?

Sounds good!

Reply:
For face intersections: [44c3a5b](https://github.com/dealii/dealii/pull/15165/commits/44c3a5bfae02f9a626b5d079df1455dec98d2e16)